lib/health_cards/health_card.rb

```diff
@@ -47,7 +47,7 @@ def compress_payload(payload)
       # @param attributes [Array] An array of string with the attribute names that will be passed through
       #  when data is minimized
       def allow(klass, attributes)
-        @allowable ||= {}
+        allowable
```

Comment:
You don't need this line, you just need to use `allowable` below instead of `@allowable`. Same thing with `types` in `#additional_types`.

Reply:
Isn't that what it's doing now?

Reply:
```ruby
def allow(klass, attributes)
  resource_type = klass.name.split('::').last
  allowable[resource_type] = attributes
end

def additional_types(*add_types)
  types += add_types
end
```

Reply:
This hasn't been addressed.

Reply:
Oops, I didn't see your follow up comment. This should be fixed now.